movement-migration/post-move2-upgrade/scripts/after-move2-upgrade.move

```diff
@@ -22,14 +25,14 @@ script {
         // Formula: rewards_amount = stake * (rewards_rate / rewards_rate_denominator) * (successful_proposals / total_proposals)
         //
         // For 10% APY with 2-hour epochs (4,380 epochs per year):
-        // Per-epoch rate = (1.10)^(1/4380) - 1 ≈ 0.0000217 or 0.00217%
+        // Per-epoch rate = (1.10)^(1/4380) - 1 ≈ 0,000021761
         //
-        // Using high precision: 22 / 1,000,000 = 0.0022% per epoch
+        // Using high precision: Reward_rate = 0,000021761 / 100_000_000 = 2176 per epoch
```

Comment:
@apenzk can we update the MIP to reflect this.  Or have discussion on this formula.

Reply:
0.2176 bps is equivalent to APY = 10%. The value for 10% APR would be 0.2283. 

Since APY is more familiar to the public the above code looks good and i agree its better to state 10% APY

I will update the MIP

cc @musitdev